mobile/lib/ui/viewer/people/people_page.dart

```diff
@@ -243,7 +241,7 @@ class _PeoplePageState extends State<PeoplePage> {
                   ],
                 );
               } else if (snapshot.hasError) {
-                log("Error: ${snapshot.error} ${snapshot.stackTrace}}");
+                _logger.severe("Error: ${snapshot.error} ${snapshot.stackTrace}}");
```

Comment:
While logging, we should prefer standard syntax _logger.server("message", e, s);